modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -13792,6 +13792,8 @@ refreshing-folder=Refreshing folder...
 refund=Refund
 refunded=Refunded
 regex=Regex
+regenerate-preview-and-thumbnail-pdf-files-for-documents-and-media=Regenerate preview and thumbnail pdf files for documents and media.
+regenerate-preview-and-thumbnail-pdf-files-for-documents-and-media-help=This action does not apply to current generated pdf previews.
```

Comment:
This sounds a bit strange. What about "Regenerate preview and thumbnail files of pdf documents in Documents and Media." .

The second one, I don't really undertand what it means.

Reply:
done